tests/test_jdate.py

```diff
@@ -137,3 +137,11 @@ def test_unpickle_older_date_object(self):
         with open('tests/pickled_objects/%s' % pickled_object_file, 'rb') as f:
             d = pickle.load(f)
         self.assertEqual(d, jdatetime.date(1400, 10, 11))
+
+    def test_fromisoformat(self):
+        d = jdatetime.date(day=22,month=2,year=1378)
+        jd = jdatetime.date.fromisoformat("1378-02-22")
+        self.assertEqual(jd, d)
+        self.assertRaises(ValueError, jdatetime.date.fromisoformat, "some-invalid-format")
+        self.assertRaises(TypeError, jdatetime.date.fromisoformat, 1)
```

Comment:
Please change this test like above.
Note: this test has to raise `TypeError: fromisoformat: argument must be str`